pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -3460,7 +3460,16 @@
     },
     "System.Runtime.Caching": {
       "InboxOn": {
-        "net45": "4.0.0.0"
+        "net45": "4.0.0.0",
```

Comment:
Why do we not have this list of mono/xamarin frameworks for eg System.Management?

Reply:
Oh perhaps because it's only in Mono on Windows?

Reply:
https://github.com/dotnet/corefx/blob/master/src/System.Management/pkg/System.Management.pkgproj

System.Management doesn't even have `AllXamarinFrameworks` as inbox. According to (https://apisof.net/catalog/System.Management) it is inbox in Mono from what I understand. So perhaps we should add them. 

@marek-safar @luhenry @weshaggard ?

Reply:
System.Management does not exist in Xamarin (it's one of those super Windows centric libraries).

Reply:
Then @danmosemsft you where right.